packages/kbn-es-archiver/src/actions/clean_kibana_indexes.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Client } from 'elasticsearch';
+import { ToolingLog, KbnClient } from '@kbn/dev-utils';
+
+import { cleanKibanaIndices, createStats } from '../lib';
+
+export async function cleanKibanaIndexesAction({
+  client,
+  log,
+  kbnClient,
+}: {
+  client: Client;
+  log: ToolingLog;
+  kbnClient: KbnClient;
+}) {
+  const stats = createStats('cleanKibanaIndices', log);
+  const kibanaPluginIds = await kbnClient.plugins.getEnabledIds();
+
+  await cleanKibanaIndices({ client, log, stats, kibanaPluginIds });
```

Comment:
I think I'd rather we just fix the `empty-kibana-index` action rather than a new one. I'm also unable to get this to work. https://gist.github.com/spalger/71260e41203e172226ca082e6e46840a is an update to the `empty-kibana-index` action which updates it to work with both OSS and Kibana with plugins, which I think means it would be what you're looking for.